doc/source/rllib/key-concepts.rst

```diff
@@ -196,6 +201,10 @@ allowing you to implement any architecture and computation logic.
     **An RLModule inside an EnvRunner actor**: The :py:class:`~ray.rllib.env.env_runner.EnvRunner` operates on its own copy of an
     inference-only version of the :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule`, using it only to compute actions.
 
+.. raw:: html
```

Comment:
Just checking: what is this doing and why is it necessary? Can this not be accomplished with native Sphinx behavior?

I checked and there's currently no use of this in the Ray docs repo. 

If you just need a dividing line, 4 hyphens should work natively:

```
----
```

Note that I'm of the opinion that if a dividing line is necessary, information should either be presented in a table or needs an additional header.